index.html

```diff
@@ -146,19 +153,19 @@ <h2 class="title">Multi-Post<br>Stories</h2>
                     </div>                
             </article>                     
 <!--about myself section from here-->
-            <section id="myself_div">    
-                <div class="picture_container">
+            <section>    
```

Comment:
- [ ] About section is looking good, great job! I would suggest reducing the width of the left column and increasing the width of the right column, this way it will look more similar to the original template:

|Template|Your design|
|---|---|
![Captura de pantalla 2022-06-02 a la(s) 10 39 38 p  m](https://user-images.githubusercontent.com/83717285/171784336-beb741de-5753-4736-a606-efa8e71f1229.png)|![Captura de pantalla 2022-06-02 a la(s) 10 39 21 p  m](https://user-images.githubusercontent.com/83717285/171784259-3dcbe5a4-d309-49e9-97b3-f47844a0671a.png)|